codebase/che/client.go

```diff
@@ -220,6 +220,108 @@ func (cs *StarterClient) StartExistingWorkspace(ctx context.Context, workspaceNa
 	return &workspaceResp, nil
 }
 
+// GetCheServerState get che server state.
+func (cs *StarterClient) GetCheServerState(ctx context.Context) (*CheServerStateResponse, error) {
+	req, err := http.NewRequest("GET", cs.targetURL("server"), nil)
+
+	if err != nil {
+		return nil, err
+	}
+
+	cs.setHeaders(ctx, req)
+
+	if log.IsDebug() {
+		b, _ := httputil.DumpRequest(req, true)
+		log.Debug(ctx, map[string]interface{}{
+			"request": string(b),
+		}, "request object")
```

Comment:
Add information to easily identify the log and function. E.g. `dump of the request to get the che server state`